app/commands/mentor/testimonial/retrieve.rb

```diff
@@ -0,0 +1,77 @@
+module Mentor
+  class Testimonial
+    class Retrieve
+      include Mandate
+
+      # Use class method rather than constant for
+      # easier stubbing during testing
+      def self.testimonials_per_page
+        10
+      end
+
+      def initialize(mentor:,
+                     page: 1,
+                     criteria: nil, order: nil,
+                     track_slug: nil,
+                     include_unrevealed: false)
+        @mentor = mentor
+        @page = page.to_i
+        @criteria = criteria
+        @order = order || :newest
+        @track_slug = track_slug
+        @include_unrevealed = include_unrevealed
+      end
+
+      def call
+        setup!
+        filter!
+        filter_track!
```

Comment:
Myabe have `filter_track` be part of `filter`?

Reply:
Updated.